devel/gitui/Portfile

```
@@ -19,12 +19,14 @@ maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  606088a2eff949995a03a8859638b4034dab7f91 \
-                    sha256  8075e180f3b01ff0c290b690488a7628c44b4de12346e04a77d823914a48918b \
-                    size    34964445
+                    rmd160  804953ac226f1e5c231bafdfd2fcb4f68ddf198f \
+                    sha256  55a85f4a3ce97712b618575aa80f3c15ea4004d554e8899669910d7fb4ff6e4b \
+                    size    34695215
 
 build.args-append   {--features "vendor-openssl"}
 
+depends_build       path:bin/cmake:cmake
```

Lets make this `depends_build-append` rather, so as not to override the `depends_build` list that is set by the portgroups above.

```suggestion
depends_build-append \
                    path:bin/cmake:cmake
```